graphcanvas/graph_view.py

```diff
@@ -115,9 +119,9 @@ def _layout_changed(self, new):
         self._canvas.style = new
 
     def _on_hover(self, label):
-        print "hovering over:", label
+        print(u"hovering over: {}".format(text(label)))
```

Comment:
In that case, I'd expect that `u"hovering over: {}".format(label)` should be enough, both on Python 2 and Python 3. It's probably worth an extra unit test, though.